.github/workflows/publish-noir-js.yml

```diff
@@ -34,6 +36,9 @@ jobs:
       - name: Build noirc_abi
         run: yarn workspace @noir-lang/noirc_abi build
 
+      - name: Build acvm_js
```

Comment:
We are building [acvm_js](https://github.com/noir-lang/noir/blob/8640bc77380b0abea888ad8e7c3dc6bb2f0d7c7c/.github/workflows/publish-acvm-js.yml#L36) one way when publishing and another way here. Why not unify approach and benefit from cache if done in sequence.